scripts/generateCommandData.js

```diff
@@ -1,4 +1,7 @@
 const path = require('path')
+// This dependency is installed in `site/package.json`
+// eslint-disable-next-line node/no-extraneous-require
```

Comment:
Should we add some `override` in `.eslintrc.json` to omit that ESLint rule inside `site/**`?

Reply:
I think we should move this script to `site/scripts`, then we won't need the override/disable comment.

I'll open a new issue for that

Reply:
https://github.com/netlify/cli/issues/1191